LeaderboardBackend/Program.cs

```diff
@@ -189,8 +194,11 @@
     );
 
     c.SupportNonNullableReferenceTypes();
-    c.SchemaFilter<RequiredNotNullableSchemaFilter>();
     c.MapType<Guid>(() => new OpenApiSchema { Type = "string", Pattern = "^[a-zA-Z0-9-_]{22}$" });
+    c.ConfigureForNodaTimeWithSystemTextJson(jsonSerializerOptions, null, null, true, new(DateTimeZoneProviders.Tzdb)
+    {
+        Instant = Instant.FromUtc(1984, 1, 1, 0, 0),
```

Comment:
Why this date, specifically?

Reply:
There just needs to be a date to use in the "example" field of the schema, otherwise it uses the current date, which breaks the CI. So I chose literally 1984.

Reply:
I cannot believe that flew over my head